### PR TITLE
Simplified section to select the Azure plan

### PR DESCRIPTION
**Pull request by mendezcirera**
Simplified section "Selecting the Azure Plan"
Created new section "Requesting access to a Private plan"

### CODE REVIEW

File: docs/intro_installation_azure.md [hunk 1]
```diff
@@ -7,39 +7,20 @@ sidebar_label: Installing in Azure
...
-* The network interface has a network security group associated.
-* The control interface has a unique and static public IP address associated.
-
-The following image depicts a graphical representation of the infrastructure elements deployed:
-
-![Conductor deployment](/img/platforms_azure_conductor_deployment.png)
+There are different Plans available for the Juniper Session Smart Networking Platform offering:
 
-#### Selecting the Azure Image
+* "Private" Plan: if there is no access to the 128 Technology repositories in the Azure environment where the software needs to be deployed (e.g., no access to the Internet), or if your preference is to avoid installing the Session Smart Networking software, a Private image can be shared with your Azure subscription via the Azure Marketplace. To request access to a Private plan, please refer to the section "Requesting access to a Private plan" <a href="https://docs.128technology.com/docs/intro_installation_azure#requesting-access-to-a-private-plan">here</a>.
```
Comment (MichaelBaj) [inline docs/intro_installation_azure.md:14]: Please refer to https://github.com/128technology/docs/blob/master/docs/CONTRIBUTING.md#hyperlinks for how to reference hyperlinks in markdown.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/intro_installation_azure.md [hunk 2]
```diff
@@ -7,39 +7,20 @@ sidebar_label: Installing in Azure
...
+* "Hourly" Plan: free trial period for 30 days, hourly software cost afterwards. This plan is recommended for Proof of Concepts and Trials only. Software upgrades and deployments outside of the cloud, e.g., on premises, require a token or certificate. The software can not be purchased via the marketplace. Refer to the Hourly plan of the **Session Smart Networking Platform** offering available [here](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/juniper-networks.session-smart-networking-payg?tab=Overview).
```
Comment (Chr1st0ph3rTurn3r) [inline docs/intro_installation_azure.md:15]: Reworded, edited look and feel, corrected link.
- **Hourly Plan:** This provides a free trial period for 30 days and an hourly software cost after the trial expires. This plan is recommended for Proof of Concepts and Trials only. Software upgrades and deployments outside of the cloud, (on premises) require a token or certificate. The software can not be purchased via the marketplace. Refer to the [Session Smart Networking Platform Hourly plan](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/juniper-networks.session-smart-networking-payg?tab=Overview) for additional information.

File: docs/intro_installation_azure.md [hunk 3]
```diff
@@ -54,27 +35,47 @@ To request access to the Private image follow the next steps:
...
 <img src="/img/platforms_azure_marketplace_image_private.png" alt="Session Smart Networking Private offering" width="192" height="243" />
 
-On the contrary, if you prefer not to use the Private image then refer to the Public image of the **Session Smart Networking** offering available [here](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/juniper-networks.session-smart-networking-payg?tab=Overview).
+## Session Smart Conductor Deployment
+
+### Requirements
+
+The following infrastructure must exist in your Azure subscription:
+* A VNet where the Conductor will be deployed.
+* The existing VNet should be segmented with at least one subnet.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/intro_installation_azure.md:56]: The existing VNet is segmented with at least one subnet.

File: docs/intro_installation_azure.md [hunk 4]
```diff
@@ -54,27 +35,47 @@ To request access to the Private image follow the next steps:
...
+  * This subnet should be reachable for SSH and HTTPs access for administration purposes.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/intro_installation_azure.md:57]: The subnet is reachable for...

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

File: docs/intro_installation_azure.md [hunk 5]
```diff
@@ -53,21 +53,21 @@ To request access to a Private plan follow the next steps:
...
-When deploying the Session Smart Conductor using the templates referenced later in this section, the following infrastructure elements are created automatically on your behalf in order to assist you with the deployment process:
-* Virtual machine using a Session Smart  image available in the marketplace.
-* The Conductor is deployed with one network interface known as the control interface.
-* The network interface has a network security group associated.
-* The control interface has a unique and static public IP address associated.
+When deploying the Session Smart Conductor using the templates referenced in this section, the following infrastructure elements are created automatically to assist with the deployment process:
+* Virtual machine using a Session Smart image available in the marketplace.
+* The Conductor is deployed with a single network interface identified as the control interface.
+** There is a network security group associated with the control interface.
+** The control interface has a unique and static public IP address.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/intro_installation_azure.md:68]: remove the extra asterisk in front of each line.

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**